static/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { Components, JSX } from './components';
+import 'quill';
```

Comment:
Does this need to be the following?
```suggestion
import { Quill } from 'quill';
```